packages/components/src/index.css

```
@@ -1,19 +1,10 @@
-@import "./styling/index.css";
-
-/* ~ is temporary as it's not ideal, see: https://github.com/gsoft-inc/sg-orbit/issues/1024 */
-@import "@hopper-ui/icons/index.css";
-@import "@hopper-ui/styled-system/index.css";
-@import "@hopper-ui/components/index.css";
+/* Must be imported first */
+@import "./core.css";
 
 /*
 NOTE: Importing all the components CSS files in a root file is not our preferred solution. Ideally, every components would import is own CSS file.
```

```suggestion
NOTE: Importing all the component CSS files in a root file is not our preferred solution. Ideally, each component should import its own CSS file.
```
Sorry I know you didn't touch this, but I saw it..